python/samples/getting_started/04-kernel-arguments-chat.ipynb

```diff
@@ -229,7 +215,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "chat_history.add_assistant_message(str(response))"
+    "chat_history += f\"<message role='assistant'>{str(response)}</message>\"\n",
```

Comment:
Same here. Turning this into a string defeats the purpose of having a `ChatHistory` object, right?

Reply:
> Same here. Turning this into a string defeats the purpose of having a `ChatHistory` object, right?

You're right. The purpose of the chat history object here is to offer convenience to construct a chat history for the model to consume. 

However, this sample is meant to show users how to use kernel argument. The chat history object here adds confusion from my fresh perspective. When I initially read the code, it's very hard to understand how this chat history object fit into the prompt, which gets passed to a chat completion API.

Reply:
I agree @moonbox3 manually inputting xml is not the getting started pattern, then we need to do a better job of explaining how ChatHistory works, not replace with something as ugly and potentially something that we will change in the future as the XML messages, the format of that XML I consider a implementation detail, not part of the API, for more advanced use cases it can be leveraged, but it has no place in getting started!